### ISSUE TITLE
[Intro to iOS] Project 2 - Rotten Tomatoes

### ISSUE THREAD

**Issue by minhkhang1795** [vuminhkhang1995/-Week-1-Assignment-1-Rotten-Tomatoes#1]
My app is complete, please review. 
/cc @codepathreview
/cc @chug2k

**Comment by chug2k** [comment]
Way to customize the launch screen! :tomato: 

I didn't mention this in class, but please be sure to follow the Swift code styling conventions for the homework. Here are a few useful guides:
- [Ray Wenderlich Swift Style Guide](https://github.com/raywenderlich/swift-style-guide)
- [Github Swift Style Guide](https://github.com/github/swift-style-guide)

Your code is pretty close to the style guide, so good job.

In class I only used a NSDictionary, but it would have been nice if you implemented a model class. The SwiftyJSON library is useful for this - https://github.com/SwiftyJSON/SwiftyJSON. 
- Nice work with the custom Movie cell.
- You shouldn't have to do `dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_LOW, 0)) {` for the URL loading, as `NSURLConnection.sendAsynchronousRequest` already runs in the background.
- Nice work loading the images asynchronously.
- Good job resizing images.
- Nice work implementing the error message on network failure, even with a close button. 
- You should also show "Network Error" for the Movie Details screen, when you try to load the high resolution image and there is no network.
- Why do you have two copies of all the code? DVDCell, MovieCell, DVDDetails, MovieDetails...this seems like too much copy and paste! 
- Only MovieDetails has a scrollview, which is great. Good work on that.
- Great work with the Grid View, although I think your images are a little stretched. How would you keep the original aspect ratio?

Great work. Your application looks very good.

**Comment by minhkhang1795** [comment]
Wow, thank you, sir! I really appreciate your comment. :) 

About the duplication of DVDCell, DVDDetails, did you mean that I should only duplicate the scenes in the StoryBoard and should not duplicate the code?

**Comment by chug2k** [comment]
yes. there are a few different ways to do it, but i am confident you could
think about it. =)
